src/style.css

```diff
@@ -0,0 +1,73 @@
+body {
+  margin: 0;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.column {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  min-width: 300px;
+  flex-basis: 33.33%;
+  flex-grow: 1;
+  color: #fff;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+}
+
+@media (min-width: 600px) {
+  .column-3 {
+    order: 1;
+  }
+
+  .column-5 {
+    order: 2;
+  }
+
+  .column-6 {
+    order: 3;
+  }
+}
+
+@media (min-width: 900px) {
+  .column-4 {
+    order: 3;
+  }
+
+  .column-5 {
+    order: 2;
+  }
+
+  .column-6 {
+    order: 1;
+  }
+}
+
+.column-1 {
+  background-color: #f00;
+}
+
+.column-2 {
+  background-color: rgb(80%, 0%, 0%);
+}
+
+.column-3 {
+  background-color: rgb(60%, 0%, 0%);
+}
+
+.column-4 {
+  background-color: rgb(40%, 0%, 0%);
+}
+
+.column-5 {
+  background-color: rgb(20%, 0%, 0%);
+}
+
+.column-6 {
+  background-color: #000;
+}
```

Comment:
stick to the same color selecting style, if you can
if you used rgb() in styles before, use it here too